src/components/Sum.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
we usually create a folder for each component and place its jsx and scss files these. E.g. components/Sum/Sum.jsx